tool/src/org/antlr/v4/codegen/model/VisitorFile.java

```diff
@@ -58,7 +58,8 @@ public VisitorFile(OutputModelFactory factory, String fileName) {
 			}
 		}
 		ActionAST ast = g.namedActions.get("header");
-		if ( ast!=null ) header = new Action(factory, ast);
```

Comment:
haha looks I or someone else already violated the rules about if statement... I guess it's OK if it's on one line like that.